Documentation/metrics.md

```diff
@@ -30,6 +30,37 @@ Pending proposal (`pending_proposal_total`) gives you an idea about how many pro
 
 Failed proposals (`proposal_failed_total`) are normally related to two issues: temporary failures related to a leader election or longer duration downtime caused by a loss of quorum in the cluster.
 
+
+### store
+
+These metrics describe the accesses into the data store of Etcd nodes that exist in the cluster (non-proxy). They 
+are useful to count what kind of actions are taken by users, and whether all Etcd nodes "see" the same set of data.
```

Comment:
I made it explicit in the description of each fields.
Made it more explicit in this sentance.